module/config.mjs

```diff
@@ -2453,14 +2454,16 @@ patchConfig("conditionTypes", "label", { since: "DnD5e 3.0", until: "DnD5e 3.2"
 /* -------------------------------------------- */
 
 /**
- * Various effects of conditions and which conditions apply it.
+ * Various effects of conditions and which conditions apply it. Either keys for the conditions,
+ * or a number for a level of exhaustion.
  * @enum {object}
  */
 DND5E.conditionEffects = {
-  noMovement: new Set(["grappled", "paralyzed", "petrified", "restrained", "stunned", "unconscious"]),
-  halfMovement: new Set(["prone"]),
+  noMovement: new Set([5, "grappled", "paralyzed", "petrified", "restrained", "stunned", "unconscious"]),
+  halfMovement: new Set([2, "prone"]),
```

Comment:
Not a big fan of just these numbers here, how do you feel about `"exhaustion-5"` instead?

```suggestion
  noMovement: new Set(["exhaustion-5", "grappled", "paralyzed", "petrified", "restrained", "stunned", "unconscious"]),
  halfMovement: new Set(["exhaustion-2", "prone"]),
```